sqrl-tools/sqrl-config/src/main/java/com/datasqrl/config/ConnectorConfigImpl.java

```diff
@@ -26,7 +26,8 @@ public class ConnectorConfigImpl implements TableConfig.ConnectorConfig {
       "filesystem", TableType.STREAM,
       "upsert-kafka", TableType.VERSIONED_STATE,
       "jdbc", TableType.LOOKUP,
-      "jdbc-sqrl", TableType.LOOKUP
+      "jdbc-sqrl", TableType.LOOKUP,
+      "datagen", TableType.STREAM
```

Comment:
anything not identified is a stream by default

Reply:
Wouldn't it be more beneficial to be explicit in this case? If not, why do we specify filesystem and Kafka? I was simply trying to follow an existing pattern. Additionally, I remember encountering an error when the connector wasn't specified.